### PR TITLE
JSON responses

### PR DESCRIPTION
**Pull request by zfletch**
This is the JSON counterpart to https://github.com/perseids-tools/morpheus-perseids-api/pull/6 (and the followup to https://github.com/perseids-tools/morpheus-perseids-api/pull/6#discussion_r387791278).

This PR:
* Builds out JSON response
* Makes JSON the default format
* Adds `respond_to` code so that the app responds with XML if the `Accept` header tells it to

### CODE REVIEW

File: lib/converter.rb [hunk 1]
```diff
@@ -42,6 +42,26 @@ class Converter
     _: '',
   }
 
+  # Taken from:
+  # https://github.com/perseids-project/morphsvc/blob/master/morphsvc/lib/transformers/LatinTransformer.py
+  LATIN_TRANSFORMS = [
+    [/[^\w\d\s]/, ''],
```
Comment (zfletch) [inline lib/converter.rb]: Like the comment notes, this is taken from the `LatinTransformer.py` file in `morphsvc`. An interesting side effect of applying these to only the Latin input is that Latin words with punctuation work but Greek words with punctuation don't work.

For example: [Greek](https://morph.perseids.org/analysis/word?lang=grc&engine=morpheusgrc&word=%CF%80%CF%8C%CE%BD%CF%84%E1%BF%B3?) and [Latin](https://morph.perseids.org/analysis/word?lang=lat&engine=morpheuslat&word=fero?).

Comment (balmas) [reply lib/converter.rb]: that's interesting. I see that in the Alpheios version too, and can't understand why it would be.

Comment (zfletch) [reply lib/converter.rb]: Looking at the regex (`r'[^\w|\d|\s]'`) again this morning, I realized that the Alpheios version does not remove the `|` character. I assume this is a bug and the regex and it should be `r'[^\w\d\s]'`.

Comment (balmas) [reply lib/converter.rb]: ah, yes that does appear to be a bug in the regex -- the | should not be there. Thanks!

Comment (balmas) [reply lib/converter.rb]: so now it makes sense to me about the punctuation

Comment (zfletch) [reply lib/converter.rb]: Made a pull request here: https://github.com/alpheios-project/morphsvc/pull/37

**Comment by zfletch** [comment]
@balmas thanks for taking a look!

**Pull request merged**